src/ModelAnalyzer.jl

```diff
@@ -102,11 +112,82 @@ function summarize(data::AbstractData; kwargs...)
     return summarize(stdout, data; kwargs...)
 end
 
+"""
+    value(issue::AbstractIssue)
+
+Return the value associated to a particular issue. The value is a number
+with a different meaning depending on the type of issue. For example, for
+some numerical issues, it can be the coefficient value.
+"""
+function value(issue::AbstractIssue, ::Nothing)
+    return value(issue)
+end
```

Comment:
```suggestion
end

```

Pet Peeve. Please add blank lines between methods.